### DISCUSSION TITLE
How to fix lints errors in model yaml file?

### DISCUSSION THREAD

**Discussion by Frank1016** [serverpod/serverpod#1892] [Q&A] [answered]
I defined a user class with relation to userInfo in the auth module the same way as in the docs:
![image](https://github.com/serverpod/serverpod/assets/32797520/c6e70fe5-53f9-46db-8d1f-c1e5bff87e2a)
However, I see immdiately these two lints errors in my yaml file in vs code:
![Screenshot 2024-02-08 at 12 15 15 PM](https://github.com/serverpod/serverpod/assets/32797520/6151d5af-b804-4e01-a3f1-f403c142bee8)
![Screenshot 2024-02-08 at 12 15 40 PM](https://github.com/serverpod/serverpod/assets/32797520/02774b3b-da85-4fe0-afaf-766b4e595e0a)
Although, after running "serverpod generate" command, it seems the relationship will be created correctly, but the errors won't go away, which is really annoying.  Does anyone know how to make these errors disappear in vs code? Thanks in advance!

**Comment by Isakdl** [comment, marked as answer]
If the generate command works without giving any errors then it should be valid in VSCode too. Did you add the module dependency after starting VSCode? If so try to reload VSCode as the plugin will load all dependencies on boot.

**Reply by Frank1016** [reply]
Indeed, a reboot of VSCode made the errors gone!
Thank you very much for the help!
